### PR TITLE
sys-fs/gt5: bump version (taken from debian), port to eapi 7

### PR DESCRIPTION
**Pull request by bearsh**
use debian source package since it contains many additions to upstream's
v1.4. also use patches from debian

### CODE REVIEW

File: sys-fs/gt5/gt5-1.5.0_pre20111220.ebuild [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2018 Gentoo Foundation
```
Comment (mgorny) [inline sys-fs/gt5/gt5-1.5.0_pre20111220.ebuild]: Please get yourself a new calendar.

File: sys-fs/gt5/gt5-1.5.0_pre20111220.ebuild [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="a diff-capable 'du-browser'"
+HOMEPAGE="http://gt5.sourceforge.net/"
+SRC_URI="http://deb.debian.org/debian/pool/main/g/${PN}/${PN}_1.5.0~20111220+bzr29.orig.tar.gz -> ${P}.tar.gz"
```
Comment (mgorny) [inline sys-fs/gt5/gt5-1.5.0_pre20111220.ebuild]: Do some substitution-foo to get this out of `${PV}`. I suppose appending `_p29` as a replacement for `bzr29` would be nice too.

File: sys-fs/gt5/gt5-1.5.0_pre20111220.ebuild [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="|| ( www-client/links
+		www-client/elinks
+		www-client/lynx )"
+
+PATCHES=(
+	"${FILESDIR}/10-bug-758634.patch"
+        "${FILESDIR}/20-bug-758645.patch"
```
Comment (mgorny) [inline sys-fs/gt5/gt5-1.5.0_pre20111220.ebuild]: Please tab-indent.

File: sys-fs/gt5/gt5-1.5.0_pre20111220.ebuild [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+)
+
+src_unpack() {
+	default
+	mv gt5* ${P}
```
Comment (mgorny) [inline sys-fs/gt5/gt5-1.5.0_pre20111220.ebuild]: Sounds like you need to override `S` to point to the correct directory instead.

**Comment by bearsh** [comment]
@mgorny thanks for reviewing! new iteration pushed

**Comment by bearsh** [comment]
@flappyports I saw you bumped gt5 to eapi7, any reason you didn't consider merging this pull request?

**Comment by juippis** [comment]
Sorry that one hasty guy failed to see your contribution :\ 

If you still have anything to add, or would like to become the proxy-maintainer, please open a new PR.

**Pull request closed without merging**